cleverhans/utils_keras.py

```diff
@@ -138,7 +139,16 @@ def _get_logits_name(self):
         """
         softmax_name = self._get_softmax_name()
         softmax_layer = self.model.get_layer(softmax_name)
-        node = softmax_layer.inbound_nodes[0]
+
+        if hasattr(softmax_layer, 'inbound_nodes'):
+          warnings.warn(
+              "Please update your version to keras >= 2.1.3; "
+              "future verions of CleverHans may drop support for this version"
```

Comment:
Could you add a 6 month deprecation warning instead? They are easier to keep track of